index.js

```diff
@@ -17,7 +17,18 @@ const manageRecipes = async () => {
 
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
-
+    
+    await Recipe.create({title: "Kachapuri", cuisine: "Georgian"})
+    await Recipe.insertMany(data)
+    console.log(data)
+
+    let message = await Recipe.findOneAndUpdate({title: 'Rigatoni alla Genovese'}, {duration: 100 }, {new: true})
+    console.log("Changes were succesfull " + message.duration)
+
+    let message2 = await Recipe.deleteOne({title: "Carrot Cake"}, {new: true})
```

Comment:
You don't need to set the `new` option to `true` here. It's used with the update function and returns the document after the update was applied.